deploy/load-test/stack/chat/mysql.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.4'
+
+services:
+    mysql-chat:
```

Comment:
Rename to `context-name-storage-name`, in this case `chat-mysql`. This applies to all service names.